src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java

```diff
@@ -70,6 +76,15 @@ public boolean isBypassable(Player p){
 		return gp.isAccessible(type, PermissionType.BLOCKS);
 	}
 	
+	public int getDamageMultiplier(){
+		Timestamp ts = NameAPI.getGroupManager().getTimestamp(g.getName());
+		
+		long shiftMultiplier = ((System.currentTimeMillis() - ts.getTime()) / (long)86400000) / (long)Citadel.getReinforcementManager().getDayMultiplier();
+		if(shiftMultiplier > 0)
+			return 2 << shiftMultiplier;
```

Comment:
Shouldn't this be shifted by shiftMultiplier - 1? Otherwise a shiftMultiplier of 0 will return a damage multplier of 1 and a shiftMultiplier of 1 will return 4